hirola/front/templates/front/change_email_sent.html

```diff
@@ -1,12 +1,64 @@
 {% extends "front/base.html" %}
 {% block styles %}
 {{ block.super }} {% load static %}
+<link rel="stylesheet" type="text/css" href="{% static 'front/css/signup_email_sent.css' %}" />
```

Comment:
If the signup_email_sent.css file is being used on another html file then changing that static file will have effects on other pages. I think it is better we create a base_xxxxxxxx.css file that has code for both then import the file on each of the files. Any html specific static file code can then be imported. That way the base prefix will enable a future dev to realise he may affect another file.

Reply:
Agreed